hermes-common/src/test/java/pl/allegro/tech/hermes/common/message/converter/AvroToJsonConverterTest.java

```diff
@@ -0,0 +1,30 @@
+package pl.allegro.tech.hermes.common.message.converter;
+
+import org.junit.Before;
+import org.junit.Test;
+import pl.allegro.tech.hermes.test.helper.avro.AvroUser;
+
+import java.io.IOException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class AvroToJsonConverterTest {
+
+    private AvroToJsonConverter avroToJsonConverter;
+    private AvroUser avroUser;
+
+    @Before
+    public void setup() throws IOException {
+        avroUser = new AvroUser();
+        avroToJsonConverter = new AvroToJsonConverter(avroUser.getSchema());
+    }
+
+    @Test
+    public void shouldConvertAvroToJson() throws IOException {
+        //when
+        byte [] json = avroToJsonConverter.convert(avroUser.create("Bob", 50, "blue"));
+
+        //then
+        assertThat(new String(json)).isEqualTo("{\"name\":\"Bob\",\"age\":50,\"favoriteColor\":\"blue\"}");
+    }
```

Comment:
Maybe we should compare contents of this JSON objects instead of their String serialized form because such a comparison is sensitive to field ordering and white symbols... Any modern object mapper should be able to do that. With that being said there is a library dedicated just for this: https://github.com/skyscreamer/JSONassert